pallets/proposals/src/lib.rs

```diff
@@ -693,6 +608,7 @@ pub mod pallet {
 
         /// Finalise a "vote of no condidence" round.
         /// Votes must pass a threshold as defined in the config trait for the vote to succeed.
+        #[transactional]
```

Comment:
I think this can be removed as we create a transaction layer in send_refund_message_to_treasury() so this is just extra overhead